### PR TITLE
Added Youtube Video

### PR DESCRIPTION
**Pull request by Eddayavuz**
I added the Youtube video under the Task title.

**This pull request makes the following changes:**
* Fixes issue #47 

**Checklist:**
- [✔️] File or folder now contains changes as specified in the issue I worked on
- [✔️] I have linked the issue I worked on to this pull request submitted by me
- [✔️] I certify that I ran my checklist.

Ping @Ifycode/git-github-training-resource

### CODE REVIEW

File: doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md
Comment (Ifycode) [inline doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md:9]: Nice try @Eddayavuz 
But this will not show up properly on gitbook, but you got the idea.
Please see embed examples in this file: https://github.com/Ifycode/git-github-training-resource/blob/develop/doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md
This should help you get the embed right.

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md
Comment (Ifycode) [reply doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md:9]: Instructions about what your pull request title should look like can be found here: https://obiagba-mary.gitbook.io/git-and-github-training/contributors-guide/making-and-submitting-changes#contribution-guidelines
Please change that too so that it's easy to track your pull request.

Comment (Eddayavuz) [reply doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md:9]: Hello, @Ifycode I did the necessary changes. Hope it works!

Comment (Ifycode) [reply doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md:9]: Nicely done @Eddayavuz
Yes the embed is correct now.

Remove the `#47` in your pull request title. Once you have done that, I will approve the pull request.

**Comment by Ifycode** [comment]
Hi @Eddayavuz I'm still waiting for your response to the requested change here https://github.com/Ifycode/git-github-training-resource/pull/48#discussion_r991032786

File: doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md
Comment (Eddayavuz) [reply doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md:9]: I removed it :)

Comment (Ifycode) [reply doc/part-1-github-ui/set-2-github-ui-learning-videos-and-tasks.md:9]: Thank you. Approved!

**Review by Ifycode — APPROVED** [review]

**Comment by Ifycode** [comment]
@all-contributors add @Eddayavuz for doc

**Pull request closed without merging**
